crate/cli/src/tests/kms/xml/kmip_1_4/kmip.rs

```diff
@@ -0,0 +1,186 @@
+use cosmian_kms_client::KmsClient;
+use serial_test::serial;
+
+use crate::tests::kms::xml::runner::{
+    run_single_xml_vector_on_client as run_single_xml_vector_on_client_generic,
+    run_single_xml_vector_with_server as run_single_xml_vector_with_server_generic,
+};
+
+/// Run a single XML vector using the shared default test server (single sqlite path).
+/// `test_name` is used to namespace UID placeholder keys to avoid cross-test collisions.
+pub(crate) async fn run_single_xml_vector(test_name: &str, path: &str) {
+    run_single_xml_vector_with_server_generic(test_name, path).await;
+}
+
+pub(crate) async fn run_single_xml_vector_on_client(
+    test_name: &str,
+    client: &KmsClient,
+    path: &str,
+) {
+    run_single_xml_vector_on_client_generic(test_name, client, path).await;
+}
+
+macro_rules! xml_test_group {
+    ($name:ident, $($file:expr),+ $(,)?) => {
```

Comment:
Isn't this case covered by the macro `xml_test_group`... which could be renamed `xml_test` then?